### ISSUE TITLE
[bug] file-carver - pass block_id and block_cound as int, not string

### ISSUE THREAD

**Issue by securityclippy** [osquery/osquery#4116]
### start_carve 

post currently passes the block_count key as a string, which should be passed as an Int

### continue_careve
post currently passes the block_id key as a string, should also be passed as an int.

This may be a breaking change for some implementations that have not adhered to the json specification, so this may be a good candidate for a 3.0 release

**Comment by muffins** [comment]
Ah, that was actually intentional for the `block_id`, as we make use of GUID values for that id. We can definitely change the `block_count`, however the block_id I might need to check a few things to ensure it doesn't break our internal deployment.

**Comment by securityclippy** [comment]
@muffins are you sure about that?

From my demo work, this is a start post
```
{"block_count":"24470","block_size":"8192","carve_size":"200455168","carve_id":"22400c8e-ab37-4856-a8dc-a7e3023e784d","request_id":"","node_key":"oh43e0133ols6k1hh2gd"}
```

and the continue post ...

```
{"block_id":"2","session_id":"12345","request_id":"","data":"<b64data snip>"}
```

I would expect `carve_id` to be a string, but I'm unclear on why `block_id` would be a guid.  Is `block_id` not just a number based on `carve_size` chunked by `block_size`?  

Its definitely not the end of the world, but adhering to standard data types makes other integrations and assumptions about the data much easier.

**Comment by muffins** [comment]
Oh true dat. You are correct.

**Comment by zwass** [comment]
This is fixed going at least back to osquery 4.1.2
